DB.fs

```diff
@@ -0,0 +1,74 @@
+﻿module KinoBot.DB
+
+
+open System.IO
+open FSharp.Data
+
+
+[<Literal>]
+let votesFile = "./db/votes.json"
+[<Literal>]
+let votesSchema = "./db/votesSchema.json"
+
+
+type VotesDB = JsonProvider<votesSchema, RootName="movie">
+
+
+let clearFile (filename : string) =
+    File.WriteAllText(filename, "[]")
+
+
+let loadOrCreateFile (filename : string) =
+    try
+        VotesDB.Load(filename)
+    with :? FileNotFoundException ->
+        clearFile filename
+        [| |]
+
+
+let currentMoviesForVote () =
+    loadOrCreateFile votesFile
+
+
+let writeVoteMovies (movies : seq<VotesDB.Movie>) =
+    let formattedMovies = movies
+                            |> Seq.map (fun movie -> movie.JsonValue.ToString())
+                            |> String.concat ","
+    let resultString = sprintf "[%s]" formattedMovies
+    File.WriteAllText(votesFile, resultString)
+
+
+(*
+    Adds a movie to the list of movies users can vote for
+    Returns true if the movie is not already present, otherwise false
+*)
+let addVoteMovie (title : string, id : string) =
```

Comment:
functions in F# are generally curried and you don't really need to specify the type if the compiler can resolve it by itself, so this would be more idiomatic like `let addVoteMovie title id =`